src/pages/what-is-ebpf.js

```diff
@@ -105,6 +105,12 @@ class Description extends React.Component {
             becoming contributors to major eBPF projects, eBPF.io will help you
             along the way.
           </p>
+	  <h3>What do eBPF and BPF stand for?</h3>
+          <p>
+            eBPF is no longer an acronym for anything and now just refers to
```

Comment:
We could give a bit more reasoning. How about this: 

BPF originally stood for Berkeley Packet Filter, but now that eBPF (extended BPF) can do so much more than packet filtering, the acronym no longer makes sense. eBPF is now considered a standalone term that doesn’t stand for anything. 

In the Linux source code, the term BPF persists, and in tooling and in documentation, the terms BPF and eBPF are generally used interchangeably. 

The original BPF is sometimes referred to as cBPF (classic BPF) to distinguish it from eBPF.

Reply:
> We could give a bit more reasoning. How about this:
> 
> BPF originally stood for Berkeley Packet Filter, 

The BSC discussed this previously and explicitly decided NOT to say this, from my recollection.

> but now that eBPF (extended BPF) 

Similarly I got the impression that the BSC didn't want to state that `e` stood for extended

> can do so much more than packet filtering, the acronym no longer makes sense. eBPF is now considered a standalone term that doesn’t stand for anything.
> 
> In the Linux source code, 

The above phrase is not really correct since the term BPF is not limited to the Linux source code.  There are plenty of other pieces of code (not just for Linux) that use the term BPF rather than eBPF.

> the term BPF persists, and in tooling and in documentation, the terms BPF and eBPF are generally used interchangeably.
> 
> The original BPF is sometimes referred to as cBPF (classic BPF) to distinguish it from eBPF.

Reply:
Obviously entirely up to the BSC, but it seems strange to me to spell out the cBPF acronym but deny that eBPF stands for anything, without any history or context. Pretty sure most readers are going to think that simply can't be right! 

I get asked all the time what eBPF stands for, what the difference is between eBPF and BPF, and why both terms continue to be used. My suggested text is based on what I usually say to answer those questions.

Reply:
Some of the background to my commentary on https://github.com/ebpf-io/ebpf.io-website/pull/149 was in the same direction as this PR - that the initial intro should be "what can you achieve with it" and not get into the details of history of BPF, eBPF, cBPF on the very front page. So from that perspective I welcome pushing these details down into a dedicated section.

Some other background here is that the Linux community has notions about the three terms:
- Classic BPF: The thing that other communities (eg BSD) would just call "BPF". Mostly what's outlined in the '93 paper.
- eBPF: "classic" BPF ISA plus various extensions including the ones that expose hooks, maps, etc which transform the capabilities to something vastly more powerful than the original.
- BPF: Catch-all for both approaches, but often meaning eBPF in Linux circles. Likely means "classic" BPF in other circles. 

For the most part I think the topic of "what does eBPF stand for?" doesn't serve much purpose other than trivia. Yes, it has roots in Berkeley Packet Filter. But it's changed enough that knowledge or understanding of that will not provide enough basis to reason about eBPF's capabilities. Using the phrasing of recent Mastodon memes, does it matter if the `Flerp` is a new version of `Shlorp` if you don't know what either `Flerp` or `Shlorp` are?

I think @lizrice's suggestion is an improvement on the current text. I do get a sense that it may be trying to discourage the use of "BPF" through wording like `originally stood` and `the term BPF persists`. In both Linux and non-Linux communities I expect that people will continue to use the BPF term, even if it is potentially ambiguous between the underyling technoligies, partly just because it's easier to say than eBPF. I'm not sure we necessarily need to nudge people in any particular direction in this regard. But I'm also not sure I would be able to do any better on the wordsmithing than that proposal.